src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -173,6 +262,25 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
         },
         (_error) => {}
       );
+    setTimeout(() => {
+      this.setConfigValues();
+    }, 1500);
```

Comment:
why this manual timeout?

Reply:
removed, will find another way to set the config values